### ISSUE TITLE
GStreamer could not create the element: gsettingsaudiosink

### ISSUE THREAD

**Issue by MiguelTremblay** [clementine-player/Clementine#5378]
### System information

- Operating System: Kubuntu Xenial (16.04)
- Clementine version: Version 1.2.3

### Expected behaviour / actual behaviour

After the update to Ubuntu Xenial Xerus, Clementine does not want to play music any more. A dialog with this error pops up :
> GStreamer could not create the element: gsettingsaudiosink. Please make sure that you have installed all necessary GStreamer plugins (e.g. OGG and MP3)

This has already been reported by others. See:
http://askubuntu.com/questions/761778/clementine-does-not-play-after-update-to-xenial/770857

#### Workaround 

After investigation, the problem comes from the Clementine config file "_$HOME/.config/Clementine/Clementine.conf_"

To fix the problem, it is possible to either erase the file, or if one wants to keep its seetings, simply remove the _GstEngine_ section within the file. Here's what mine was looking like:
> [GstEngine] 
> sink=gsettingsaudiosink 
> device= 
> rgenabled=false 
> rgmode=0 
> rgpreamp=@Variant(\0\0\0\x87\0\0\0\0)
> rgcompression=true
> bufferduration=1000 
> monoplayback=false

**Comment by sithembiso** [comment]
Anyone else having this issue? I tried the above, but I don't have sound still.

**Comment by Solution** [comment]
Yep, the same issue here, erasing the gstengine section didnt work, erasing complete file also didnt work.

**Comment by MiguelTremblay** [comment]
If you remove Clementine.conf file completly amd restart Clementine, does it resolve this issue?

**Comment by themaddoctor** [comment]
No, it doesn't

**Comment by tommyjcarpenter** [comment]
I am also having this issue on Arch Linux (fresh isntall). I had `gstreamer` installed. I fixed this issue with two things:
1) installing moar plugins: `yo -S gst-plugins-good gst-plugins-base gst-libav gst-plugins-bad gst-plugins-ugly`
2) deleting the Clementine.conf file and rebooting and restarting clementine

It did not work when I did either of these alone.

**Comment by gleporeNARA** [comment]
Had the problem after re-installed to KDE Neon 64 bit and copying the old 32 bit .conf file over. Deleting the conf file and restart fixed the issue.
